policy/policy_handler/mocks/image_packages_by_digest.go

```diff
@@ -0,0 +1,79 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/atomist-skills/go-skill"
+	"github.com/atomist-skills/go-skill/policy/policy_handler/legacy"
+	"github.com/atomist-skills/go-skill/policy/types"
+)
+
+func MockImagePackagesByDigest(ctx context.Context, req skill.RequestContext, sb *types.SBOM) (legacy.ImagePackagesByDigestResponse, error) {
+	req.Log.Info("Building local evaluation mocks for image packages by digest")
+
+	if len(sb.Vulnerabilities) == 0 {
+		req.Log.Info("SBOM doesn't provide any vulnerabilities directly, fetching them from GraphQL")
+
+		var pkgs []legacy.Package
+		for _, a := range sb.Artifacts {
+			pkgs = append(pkgs, legacy.Package{
+				Type:      a.Type,
+				Namespace: a.Namespace,
+				Name:      a.Name,
+				Version:   a.Version,
+				Purl:      a.Purl,
+				Licenses:  a.Licenses,
+			})
+		}
+		res, err := legacy.MockImagePackagesByDigest(ctx, req, pkgs)
```

Comment:
Would be nice to have this functionality not in `legacy` since it isn't really legacy, but not a big deal.